torch/csrc/jit/serialization/export.cpp

```diff
@@ -827,12 +827,7 @@ std::string pretty_print_onnx(
   return prettyPrint(graph_encoder.get_model_proto());
 }
 
-// export_raw_ir will export IR ops without turning them into ONNX ops.
-// The output will use the ONNX protobuf format, but the ops will not
-// conform to the ONNX op specification. Thus, the output will not
-// be interpretable by a ONNX-compatible framework. However, PyTorch or
-// libtorch will be able to import the IR and play it back.
-std::tuple<std::string, RawDataExportMap> export_onnx(
+std::tuple<::ONNX_NAMESPACE::ModelProto, RawDataExportMap> _export_onnx(
```

Comment:
I'm wondering if we could use template type taking the place of `ModelProto`, and with `type_trait` determine if it should return `string` or `ModelProto`.